src/Processors/Formats/Impl/ValuesBlockInputFormat.cpp

```diff
@@ -139,6 +139,29 @@ void ValuesBlockInputFormat::readRow(MutableColumns & columns, size_t row_num)
     ++total_rows;
 }
 
+bool ValuesBlockInputFormat::tryReadDefaultValue(IColumn & column, size_t column_idx)
+{
+    try
+    {
+        if (!checkStringByFirstCharacterAndAssertTheRestCaseInsensitive("DEFAULT", *buf))
```

Comment:
Why `checkStringByFirstCharacterAndAssertTheRestCaseInsensitive` that may throw exception is used? Seems like we can simple try to find `DEFAULT` in buffer and if it was not found then simply call `rollbackToCheckpoint()` without try-catch

Reply:
(I see that it was initially implemented that way in #33141, but looks like #33141 was merged by mistake)